awx/ui/client/src/credentials/factories/kind-change.factory.js

```diff
@@ -164,19 +163,6 @@ export default
                 scope.authorize = false;
                 scope.authorize_password = null;
             }
-
-            // Collapse or open help widget based on whether scm value is selected
```

Comment:
not sure how this works now this code is removed.

Reply:
To be honest - we might be able to get rid of this entire factory.  The only place it's used is in the adhoc controller when the user hits the reset button and looking at the code for this factory I'm not sure that it actually does anything useful